skimage/morphology/footprints.py

```diff
@@ -174,6 +298,7 @@ def _decompose_size(size, kernel_size=3):
     return 1 + (size - kernel_size) // (kernel_size - 1)
 
 
+@deprecate_func(deprecated_version="0.25", removed_version="0.27", hint="Use `skimage.morphology.footprint_rectangle` instead.")
```

Comment:
Should we modify tests to rather test `footprint_rectangle` instead?

Reply:
Yes, definitely. This is work in progress and I'm not done at all here! Sorry, I could have worded that clearer in the PR description.

Reply:
Overzealous reviewer; not often that happens :)

Reply:
Done.